### PR TITLE
751: Adds sentry integration to client

### LINKED ISSUE #751: Implement Sentry in Hedwig Client Source
Sentry has been implemented in the Hedwig backend and needs to be implemented in the Hedwig client.  

For more information on setting up Sentry in the Hedwig client source see:
* https://sentry.io/skylight-digital/hedwig-fe-staging/getting-started/javascript-react/

The change to source code to support Sentry in the Hedwig Client must not be executed in the docker-based run-times used for local development and PR systems.

Once the client source has been modified to support Sentry, the Hedwig Release pipelines need to be updated to embed Sentry environment (e.g. Staging, UAT, Production) data in the client source.  Updating the Release pipelines will be tracked in another issue .

### PR DESCRIPTION
**Pull request by jlhogan**
Sentry needs to be setup within the Hedwig react front end. This integration is highly documented in articles like:

* https://docs.sentry.io/platforms/javascript/react

The Hedwig react code, the Sentry properties will be in the public/config.json file as follows:
````
{
        "Sentry.FEDsn": "",
        "Sentry.Release": ""
}
````
This issue closes #751

### CODE REVIEW

File: src/Hedwig/Hedwig.csproj [hunk 1]
```diff
@@ -46,6 +46,12 @@
 		<None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Update="ClientApp\.env">
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+	  </None>
+	</ItemGroup>
+
```
Comment (cailyncodes) [inline src/Hedwig/Hedwig.csproj]: This can probably be removed

File: src/Hedwig/ClientApp/src/containers/Enrollment/Edit/EnrollmentEdit.test.tsx [hunk 2]
```diff
@@ -30,33 +30,33 @@ const history = createBrowserHistory();
...
-
-			const firstNameErr = await waitForElement(() =>
-				getByText('This information is required for enrollment')
-			);
-
-			expect(firstNameErr).toBeInTheDocument();
-			// TODO: the id needs to be updated-- this will probably mess things up
-			expect(firstNameErr.id).toBe('child-firstname-error');
-		});
+//		it('shows an error if rendered without a child first name', async () => {
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/containers/Enrollment/Edit/EnrollmentEdit.test.tsx]: Should uncomment this

File: azure-pipelines.yml [hunk 3]
```diff
@@ -165,15 +165,15 @@ jobs:
   # Publish artifact to dropbox - hedwig-spa
   - task: PublishBuildArtifacts@1
     displayName: "Publish Artifacts: hedwig-spa"
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    #condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```
Comment (cailyncodes) [inline azure-pipelines.yml]: Note to revert before merging

Comment (cailyncodes) [reply azure-pipelines.yml]: And on line 176

File: src/Hedwig/Hedwig.csproj (hunk 1 above)
Comment (speedcm) [reply src/Hedwig/Hedwig.csproj]: This code stanza was removed.

File: azure-pipelines.yml (hunk 3 above)
Comment (speedcm) [reply azure-pipelines.yml]: This issue is now resolved.

**Comment by speedcm** [comment]
The implementation of Sentry in the DevOps tools to support the Hedwig client is document at https://github.com/skylight-hq/ctoec-devops/blob/master/docs/TOOLS-SENTRY.md

**Comment by speedcm** [comment]
The Staging and UAT Release Pipelines have been updated to support the Sentry implementation for Hedwig client.  After this PR is merged to master, the Production Release Pipeline must also be updated.

**Pull request merged**